init.js

```diff
@@ -1,7 +1,29 @@
 'use strict'
 
-const tracer = require('.')
+const path = require('path')
+const Module = require('module')
 
-tracer.init()
+let initBailout = false
 
-module.exports = tracer
+if (process.env.DD_INJECTION_ENABLED) {
```

Comment:
What are the possible values for this? Could it be `0` or `false`?

Reply:
Env vars are always strings, of course. IIUC the only possible values are unset, `"tracing"`, and `"tracing,service_name"`.